install.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 
 if [[ -z $OPOSSUM_HEADLESS_SETUP ]]; then
+    BOOST_INSTALLED=$(dpkg-query -W --showformat='${Status}\n' libboost1.67-dev|grep "install ok installed")
+    if [ "" != "$BOOST_INSTALLED" ]; then
+         read -p 'libbost1.67-dev is installed. Deinstall to avoid errors? [y|n] ' -n 1 -r < /dev/tty
```

Comment:
libbost1.67-dev is installed, 1.70 is required. Ok to remove 1.76-dev?

Reply:
copy pasta